caddyfile.go

```diff
@@ -288,6 +288,11 @@ func parseCaddyfileTokenValidator(h httpcaddyfile.Helper) (caddyhttp.MiddlewareH
 					return nil, h.Errf("%s argument has no value", rootDirective)
 				}
 				p.UserIdentityField = h.Val()
+			case "use_js_redir":
```

Comment:
@caiych , please move this one under `case "enable":` and change the directive from `use_js_redir` to `js redirect`.

This way enabling it would mean writing something like.

```
enable js redirect
```